cores/esp8266/FS.h

```diff
@@ -54,6 +54,12 @@ class File : public Stream
     // Print methods:
     size_t write(uint8_t) override;
     size_t write(const uint8_t *buf, size_t size) override;
+    // These handle ambiguity for write(0) case
+    size_t write(int8_t t) { return write((uint8_t)t); }
```

Comment:
Do we want these here or in the Print class?
Given that HardwareSerial, which inherits from Stream, and therefore Print, I think these should be put in Print, and HardwareSerial should also be cleaned up, because the write methods there do pretty much the same thing.

Reply:
Done for both.  Seems to build fine in Arduino and host tests.